packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+
+    t.like(readLatest('published.wallet.agoric1buyer'), {
+      status: {
+        id: 'bid1',
+        result: 'Your bid has been accepted',
+        payouts: undefined,
+      },
+    });
+    await buyer.sendOfferMaker(Offers.auction.Bid, {
+      offerId: 'bid2',
+      give: setup.bids[1].give,
+      price: setup.bids[1].price,
+      maxBuy,
+      parseAmount,
+    });
+    t.like(readLatest('published.wallet.agoric1buyer'), {
+      status: {
+        id: 'bid2',
+        result: 'Your bid has been accepted',
+        payouts: undefined,
+      },
+    });
+    await buyer.sendOfferMaker(Offers.auction.Bid, {
+      offerId: 'bid3',
+      give: setup.bids[2].give,
+      discount: setup.bids[2].discount,
+      maxBuy,
+      parseAmount,
+    });
+  }
+  t.like(readLatest('published.wallet.agoric1buyer'), {
+    status: {
+      id: 'bid3',
+      result: 'Your bid has been accepted',
+      payouts: undefined,
+    },
+  });
+
+  {
+    // ---------------
+    //  Change price to trigger liquidation
+    // ---------------
+
+    await priceFeedDriver.setPrice(9.99);
+
+    // check nothing liquidating yet
+    /** @type {import('@agoric/inter-protocol/src/auction/scheduler.js').ScheduleNotification} */
+    const liveSchedule = readLatest('published.auction.schedule');
+    t.is(liveSchedule.activeStartTime, null);
+    t.like(readLatest('published.vaultFactory.managers.manager0.metrics'), {
+      numActiveVaults: setup.vaults.length,
+      numLiquidatingVaults: 0,
+    });
+
+    // advance time to start an auction
+    await advanceTimeTo(NonNullish(liveSchedule.nextDescendingStepTime));
+    t.like(readLatest('published.vaultFactory.managers.manager0.metrics'), {
+      numActiveVaults: 0,
+      numLiquidatingVaults: setup.vaults.length,
+      liquidatingCollateral: {
+        value: scale6(setup.auction.start.collateral),
+      },
+      liquidatingDebt: { value: scale6(setup.auction.start.debt) },
+    });
+
+    console.log('step 1 of 10');
+    await advanceTimeBy(3, 'minutes');
+    t.like(readLatest('published.auction.book0'), {
+      collateralAvailable: { value: scale6(setup.auction.start.collateral) },
+      startCollateral: { value: scale6(setup.auction.start.collateral) },
+      startProceedsGoal: { value: scale6(setup.auction.start.debt) },
+    });
+
+    console.log('step 2 of 10');
+    await advanceTimeBy(3, 'minutes');
+
+    console.log('step 3 of 10');
+    await advanceTimeBy(3, 'minutes');
+    // XXX updates for bid1 and bid2 are appended in the same turn so readLatest gives bid2
+    // NB: console output shows 8897786n payout which matches spec 8.897ATOM
+    // t.like(readLatest('published.wallet.agoric1buyer'), {
+    //   status: {
+    //     id: 'bid1',
+    //     payouts: {
+    //       Bid: { value: 0n },
+    //       Collateral: { value: scale6(outcome.bids[0].payouts.Collateral) },
+    //     },
+    //   },
+    // });
+    readLatest('published.wallet.agoric1buyer.current');
```

Comment:
what does this `readLatest` call do?

Reply:
when the tracer is on it logs to console. I'll remove